syntax = "proto3";

service ReportService {
  rpc GetReport(ReportId)returns(Report){}
}


message ReportId{
  int32 Id = 1;
}

message Report {
  string OrderName = 1;
  int32  OrderId =2;
  string UserName = 3;
  string ProductName = 4;
  int32  Count = 5;
}